articles/active-directory/develop/tutorial-v2-asp-webapp.md

```diff
@@ -473,11 +473,11 @@ Learn about Web apps calling web APIs:.
 ### Learn the steps to create the application used in this quickstart
 
 > [!div class="nextstepaction"]
-> [Web apps calling Web APIs]( https://aka.ms/msal-net-authorization-code)
+> [Web apps calling Web APIs](https://docs.microsoft.com/en-us/graph/tutorials/aspnet)
```

Comment:
I would rather propose a link to the scenario landing page, and, then a link to the Microsoft Graph tutorial:

```md
> [!div class="nextstepaction"]
> [Web apps calling Web APIs](scenario-web-app-sign-user-overview.md)

> [!div class="nextstepaction"]
> [Microsoft Graph ASP.NET tutorial](https://docs.microsoft.com/en-us/graph/tutorials/aspnet)
```